client/src/components/IndexPage.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+const IndexPage = (props) => {
+  const [trails, setTrails] = useState([]);
+
+  const getTrails = async () => {
+    try {
+      const response = await fetch("/api/v1/index");
```

Comment:
we should be fetching `api/v1/trails` because `trails` is the resource we're accessing - we don't have a resource called index